src/PayPal/Provider.php

```diff
@@ -67,30 +67,24 @@ protected function mapUserToObject(array $user)
         return (new User())->setRaw($user)->map([
             'id'       => str_replace('https://www.paypal.com/webapps/auth/identity/user/', null, $user['user_id']),
             'nickname' => null, 'name' => $user['name'],
-            'email'    => $user['email'], 'avatar' => null,
+            'email'    => !empty($user['emails']) ? $user['emails'][0]['value'] : null,
```

Comment:
Should we not use the primary email?  code below not tested

```suggestion
            'email'    => collect($user['emails'] ?? [])->firstWhere('primary', true),
```

Reply:
Yea fair point, changed in latest commit